source/community/reactnative/src/components/containers/factset.js

```diff
@@ -137,6 +137,7 @@ export class FactSet extends React.Component {
 	}
 
 	render() {
+		this.payload = props.json;
```

Comment:
should this be this.props.json instead? @frags51

Reply:
Yes, good catch.

Reply:
Changed